witchcraft-server/src/logging/mod.rs

```diff
@@ -42,6 +42,8 @@ mod trace;
 
 pub(crate) static AUDIT_LOGGER: AtomicLazyCell<Arc<Mutex<Appender<AuditLogV3>>>> =
     AtomicLazyCell::NONE;
+pub(crate) static EVENT_LOGGER: AtomicLazyCell<Arc<Mutex<Appender<EventLogV2>>>> =
+    AtomicLazyCell::NONE;
```

Comment:
Copied the mutex idea from the audit logger, assuming this was done because the global function could be used in multiple async contexts or something, so figured the same principle would apply here - but let me know if that's not correct/overkill in this context

Reply:
I think we can actually get rid of the mutex here once we switch over to try_send in the event_log function.